src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -475,6 +492,7 @@ def main():
 
     # A testcase is untriaged, until immediately before a bug is opened
     _emit_untriaged_testcase_age_metric(testcase)
```

Comment:
This would overcount untriaged testcases since there's a good chance the bug will be filed almost immediately after.
Can we move this call to line 500 where issue filing fails?